content/en/docs/setup/upgrade/index.md

```diff
@@ -102,6 +102,10 @@ To uninstall a control plane of revision `istio-1-6-5`, run:
 $ istioctl x uninstall --revision istio-1-6-5
 {{< /text >}}
 
+{{< warning >}}
+This command will also remove gateway deployments for specified revision. If you want to remove the canary revision, please reinstall the gateway(s) with previous revision first to avoid down time.
```

Comment:
Is this correct all the time?  I think it won't remove if I'm deleting the old revision (1-6-5) but would if i'm deleting the new revision (1-7-0).

Reply:
I propose to have 2 sections, 1 for remove old cp, 1 for remove the new version (canary).  This warning only applied to the 2nd scenario.

Reply:
I added some text for this along with another fix in #7957. If that looks alright, then we can close this PR.

Reply:
@linsun yes, that's why I emphasized `canary` one here. But if that's not clear enough, then different section is ok